sdk/swagger.yaml

```diff
@@ -427,6 +495,30 @@ components:
         application/json:
           schema:
             $ref: "#/components/schemas/OrgEnvironments"
+    ListEnvironmentTags:
+      description: List of environment tags
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/ListEnvironmentTags"
+    CreateEnvironmentTag:
+      description: Create environment tag
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/CreateEnvironmentTag"
+    UpdateEnvrionmentTag:
+      description: Update environment tag
+      content:
+        application/json:
+          schema:
+            $ref: "#/components/schemas/UpdateEnvironmentTag"
+    EnvironmentTag:
+      description: Create environment tag

```

Comment:
```suggestion
      description: Get an environment tag

```

Reply:
This is the response from create environment tag